src/SonarQube.Common/Utilities.cs

```diff
@@ -30,6 +30,11 @@ public static class Utilities
     {
         #region Public methods
 
+        private static Lazy<string> scannerVersion =>
```

Comment:
Why `Lazy`? It's pretty much guaranteed the scanner will make HTTP requests, isn't it?

Fine with leaving it, just curious.

Reply:
It seemed like a good idea, but is probably overengineering, since the first thing the scanner does is to print its version in the console...